skbio/metadata/__init__.py

```diff
@@ -16,4 +16,12 @@
 # ----------------------------------------------------------------------------
 
 from skbio.util import TestRunner
+
+from ._interval import IntervalMetadata
+from ._mixin import (PositionalMetadataMixin, MetadataMixin)
```

Comment:
Remove PositionalMetadataMixin and MetadataMixin imports, they are private classes